internal/server/server.go

```diff
@@ -59,6 +60,11 @@ func New(logger Logger, client Client, templates map[string]*template.Template,
 			allowRoles(client, "System Admin", "Manager")(
 				editTeam(client, templates["team-edit.gotmpl"], siriusURL))))
 
+	mux.Handle("/teams/add/",
+		wrap(
+			allowRoles(client, "System Admin", "Manager")(
```

Comment:
Adding a new team requires being a system admin (strictly, it requires having the PUT/team permission)
```suggestion
			systemAdminOnly(
```

Reply:
I should really read the stories